css/styles.css

```diff
@@ -73,6 +73,16 @@ html, body {
   opacity: .7;
 }
 
+.mute-audio {
+  position: fixed;
+  font-size: 1.3rem;
+  margin: 8px;
+  color: white;
+  bottom: 10%;
```

Comment:
The button is not visually discernable from text.

Reply:
Changed font family to "Arial Black"

Reply:
Just changing the font is not enough. In any case provide a screenshot. Will discuss further what to do.